ui/pages/swaps/slippage-buttons/slippage-buttons.js

```diff
@@ -133,20 +134,20 @@ export default function SlippageButtons({
                       setCustomValue('');
                       setEnteringCustomValue(false);
                       setActiveButtonIndex(0);
-                      onSelect(2);
+                      onSelect(SLIPPAGE.DEFAULT);
                     }}
                   >
-                    2%
+                    {`${SLIPPAGE.DEFAULT}%`}
```

Comment:
Should we localize this? It seems like in[ persian and turkish the percentage sign comes first](https://www.ultius.com/glossary/grammar/punctuation/percentage-sign.html#:~:text=In%20Persian%20and%20Turkish%2C%20the,written%20as%200%2F01).).